### PR TITLE
DOCK-2594: Fix TRS ID "copy button" bug

### PR DESCRIPTION
**Pull request by svonworl**
**Description**
This PR fixes the "copy" button (on the workflow page, in the "Workflow Information" section, at the right end of the "TRS" line) so that when two workflows are viewed successively, the copied value is correct.  Previously, if you visited the pages for two workflows A and B, if you used the button to copy the TRS ID on workflow B's page, the copied value was the TRS ID for workflow A.

The cause was that we were updating the TRS ID field of the component when an "entryType" observable issued forth a new value.  The assumption was probably that if we visited successive workflows, we'd see successive "workflow" entry type values on the line.  But, for whatever reason, that's not how it was working...

Editorial: this type of bug suggests that our UI code (and more specifically, the architecture and constructs typically employed within) is making it hard for us to do simple, straightforward things (such as, in this case, display the TRS ID corresponding to the current workflow).

The branch has the wrong issue number, woops.

**Review Instructions**
Go to search, reproduce the steps in the first paragraph of the "Description" section above, use the button to copy the TRS ID, and confirm it has the correct value.

**Issue**
https://ucsc-cgl.atlassian.net/browse/DOCK-2594
dockstore/dockstore#6024

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

File: src/app/workflow/info-tab/info-tab.service.ts [hunk 1]
```diff
@@ -187,11 +187,12 @@ export class InfoTabService {
     );
   }
 
-  getTRSId(workflow: Workflow | undefined, entryType: EntryType): string {
-    if (!workflow) {
+  getTRSId(workflow: Workflow | undefined): string {
+    if (workflow) {
+      return workflow.entryTypeMetadata.trsPrefix + workflow.full_workflow_path;
```
Comment (kathy-t) [inline src/app/workflow/info-tab/info-tab.service.ts:192]: No need to change, but FYI, there is a `trsId` field now:  https://github.com/dockstore/dockstore/blob/c8459501ac20132294376702c6c8a81338d3c9bf/dockstore-webservice/src/main/resources/openapi3/openapi.yaml#L12529-L12530

**Review by kathy-t — APPROVED** [review]

**Review by david4096 — APPROVED** [review]

File: src/app/workflow/info-tab/info-tab.service.ts (hunk 1 above)
Comment (kathy-t) [reply src/app/workflow/info-tab/info-tab.service.ts]: the `trsId` should be available for `Workflow` https://github.com/dockstore/dockstore/blob/c8459501ac20132294376702c6c8a81338d3c9bf/dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java#L969-L971

Comment (svonworl) [reply src/app/workflow/info-tab/info-tab.service.ts]: Forgot about that, changed.  Thanks Kathy!

Comment (svonworl) [reply src/app/workflow/info-tab/info-tab.service.ts:192]: Woops, something is wrong.  Being that I'm leaving soon and we're trying to deploy again, reverting to original solution...

**Pull request merged**